torch/csrc/autograd/init.cpp

```diff
@@ -258,12 +258,22 @@ PyObject* THPAutograd_initExtension(PyObject* _unused, PyObject* unused) {
 
   {
     using torch::profiler::impl::Result;
+    py::class_<PyFrameState>(m, "_PyFrameState")
```

Comment:
I think this is ok to expose some of these objects but maybe these should be in their own submodule and not in the autograd/profiler bindings?

Reply:
Yeah, it's probably about time to make a profiler submodule. Those are in autograd mostly out of inertia.